app/_src/policies/meshmetric.md

```diff
@@ -66,17 +66,82 @@ You can define configuration refresh interval by using `KUMA_DATAPLANE_RUNTIME_D
 ### Sidecar
 
 This part of the configuration applies to the data plane proxy scraping.
-
 In case you don't want to retrieve all Envoy's metrics, it's possible to filter them.
-You are able to specify [`regex`](https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--stats?filter=regex) which causes the metrics endpoint to return only matching metrics.
-By default, metrics that were not updated won't be published. You can set flag `includeUnused` that returns all metrics from Envoy.
+Below are different methods of filtering.
```

Comment:
shouldn't this be wrapped with `{% if_version gte:2.7.x %}`, this is not available in 2.6.x version

Reply:
let me know if it looks right now